src/xnnpack/microparams.h

```diff
@@ -170,6 +170,12 @@ union xnn_f32_minmax_params {
     XNN_ALIGN(8) float max[2];
   } wasmsimd;
 #endif  // XNN_ARCH_WASMSIMD || XNN_ARCH_WASMRELAXEDSIMD
+#if XNN_ARCH_HEXAGON
+  struct {
+    XNN_ALIGN(128) float min[128];
```

Comment:
Should this be 32 (the array size, not the alignment)? This is a 512 byte array, I think we want 128 bytes here?

Reply:
Ah yes that's a mistake should be 32